github-repo-manager/src/main/java/com/axonivy/github/GitHubStaleBranchesDetector.java

```diff
@@ -0,0 +1,77 @@
+package com.axonivy.github;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GitHubStaleBranchesDetector {
+
+  public static void main(String[] args) throws IOException {
+    var github = GitHubProvider.get();
+    var org = github.getOrganization("axonivy");
+    var repos = List.copyOf(org.getRepositories().values());
+
+    var lines = new ArrayList<LastCommit>();
+
+    for (var repo : repos) {
+      if (repo.isArchived()) {
+        continue;
+      }
+      if ("xivy-3.9".equals(repo.getName())) {
+        continue;
+      }
+
+      for (var branch : repo.getBranches().values()) {
+        if ("master".equals(branch.getName())) {
+          continue;
+        }
+        if ("gh-pages".equals(branch.getName())) {
+          continue;
+        }
+        if ("dev10.0".equals(branch.getName())) {
+          continue;
+        }
+        if ("dev11.1".equals(branch.getName())) {
+          continue;
+        }
+        if (branch.getName().startsWith("release/")) {
+          continue;
+        }
+        if (branch.getName().startsWith("stale/")) {
+          continue;
+        }
+        if (branch.getName().startsWith("dependabot/")) {
+          continue;
+        }
```

Comment:
not better to make some list with ignored bbrnaches?

Reply:
done